custom_components/tuya_local/devices/plikc_neve_x_thermostat.yaml

```diff
@@ -0,0 +1,147 @@
+name: Termostato
+products:
+  - id: 5kwgvcaqzzkzrftu
+    name: Plikc Neve X W
+primary_entity:
+  entity: climate
+  translation_key: thermostat
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          value: heat
+    - id: 2
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: auto
+          value: home
+        - dps_val: manual
+          value: boost
+        - dps_val: temporary
+          value: boost
+          hidden: true
+        - dps_val: holiday
+          value: away
+          constraint: Durata vacanza
+          conditions:
+            - dps_val: 1
+    - id: 3
+      type: string
+      name: hvac_action
+      readonly: true
+      mapping:
+        - dps_val: heat
+          value: heating
+        - dps_val: cold
+          value: cooling
+        - dps_val: "off"
+          value: idle
+    - id: 16
+      type: integer
+      name: temperature
+      unit: C
+      range:
+        min: 50
+        max: 350
+      mapping:
+        - scale: 10
+          constraint: preset_mode
+          conditions:
+            - dps_val: holiday
+              value_redirect: holiday_temp_set
+    - id: 24
+      type: integer
+      name: current_temperature
+      mapping:
+        - scale: 10
+    - id: 32
+      type: integer
+      name: holiday_temp_set
+      optional: true
+      hidden: true
+      force: true
+      range:
+        min: 50
+        max: 350
+      mapping:
+        - scale: 10
+secondary_entities:
+  - entity: number
+    name: Calibrazione
+    mode: box
+    category: config
+    icon: "mdi:thermometer-check"
+    dps:
+      - id: 27
+        type: integer
+        name: value
+        unit: °C
```

Comment:
It is better to use ° for calibration adjustments, because you don't want HA doing unit conversion on these.